safe_relay_service/tokens/urls.py

```diff
@@ -1,9 +1,19 @@
+from django.conf import settings
 from django.urls import path
 
+from safe_relay_service.tokens.models import Token
 from . import views
 
 app_name = "tokens"
 
 urlpatterns = [
     path('', views.TokensView.as_view(), name='tokens'),
 ]
+
+if settings.SAFE_DEFAULT_TOKEN_ADDRESS:
```

Comment:
Was this here before and we removed it? Seems like they should have already had handling for that, no?

Reply:
afaict they use a manage.py command to add a token. I couldn't get that to work locally (docker networking probably) so I tried this instead